.github/workflows/update-readme.yml

```diff
@@ -21,5 +21,17 @@ jobs:
       - name: Compile parsers
         run: ./nvim.appimage --headless -c "TSInstallSync all" -c "q"
 
+      # inspired by nvim-lspconfigs
       - name: Check README
-        run: ./nvim.appimage --headless -c "luafile ./scripts/update-readme.lua" -c "q"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          COMMIT_MSG: |
+            [docgen] Update README.md
+            skip-checks: true
```

Comment:
We will need to filter that, maybe rename that to `no-readme-update` and filter that in the action.

Reply:
I don't think that we need the `skip-checks: true` here.

Reply:
It doesn't have any effects right now. But usually it would skip the checks for the docgen commit (tests and recursive call to it itself, though there shouldn't be any changes then)